Cargo.toml

```diff
@@ -67,7 +69,7 @@ jsonfile_auth = ["unftp-auth-jsonfile"]
 pam_auth = ["unftp-auth-pam"]
 rest_auth = ["unftp-auth-rest"]
 tokio_console = ["console-subscriber", "tokio/tracing"]
-
+azblob = ["dep:unftp-sbe-opendal", "dep:opendal", "opendal/services-azblob"]
```

Comment:
Can opendal compile with musl? If so we can add `azblob` to the `musl` feature below.